### DISCUSSION TITLE
launch and configure kobold via python code

### DISCUSSION THREAD

**Discussion by madrooky** [LostRuins/koboldcpp#567] [Q&A] [answered]
I wrote an app that uses Kobold as endpoint to send specific information to be processed and then uses the output further to complete the task.
I would like to know if it was possible to launch the Kobold app from the same script, load a config, and close the endpoint again when all tasks are completed.

Perhaps a dumb question, but i just started coding a few days ago^^

**Comment by gustrd** [comment, marked as answer]
I believe this use case is the basic use case of the upstream llama.cpp. Give a look.
